src/frontend/src/routes/(app)/+layout.svelte

```diff
@@ -15,18 +15,16 @@
 	import { authNotSignedIn, authSignedIn } from '$lib/derived/auth.derived';
 	import { pageToken } from '$lib/derived/page-token.derived';
 	import { token } from '$lib/stores/token.store';
-	import { isRouteDappExplorer, isRouteSettings, isRouteTransactions } from '$lib/utils/nav.utils';
+	import { isRouteTokens, isRouteTransactions } from '$lib/utils/nav.utils';
 
-	// TODO: We should consider adding a description for the pages, as this block of code is now appearing in two places.
-	// Other areas, like the Menu, are also somewhat disorganized, with navigation logic spread across multiple locations.
-	let route: 'transactions' | 'tokens' | 'settings' | 'explore' = 'tokens';
-	$: route = isRouteSettings($page)
-		? 'settings'
-		: isRouteDappExplorer($page)
-			? 'explore'
-			: isRouteTransactions($page)
-				? 'transactions'
-				: 'tokens';
+	let tokensRoute: boolean;
+	$: tokensRoute = isRouteTokens($page);
+
+	let transactionsRoute: boolean;
+	$: transactionsRoute = isRouteTransactions($page);
+
+	let showHero: boolean;
+	$: showHero = tokensRoute || transactionsRoute;
```

Comment:
Out of curiosity, not a call to action: we already have E2E screenshots of every pages? That way we can assert that indeed the hero is displayed only when needed.

Reply:
I agree! But we don't have for all yet, I was working on it. I have the first PR for the transactions page already in draft:

- #3345